src/posts/2020-10-14-wcag2.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: Introduction to WCAG 2
+description: WCAG is a set of accessibility guidelines published by WAI...
+category: Background
+author: Hamsa Harcourt
+further_reading:
+  - url: https://www.w3.org/TR/WCAG20/#intro-layers-guidance 
+    title:  Web Content Accessibility Guidelines (WCAG) 2.0
+    source: W3C 
+  - url:  https://en.wikipedia.org/wiki/Web_Content_Accessibility_Guidelines
+    title: Web Content Accessibility Guidelines
+    source: Wikipedia
+date: 2020-10-14
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - background
+---
+The **[Web Content Accessibility Guidelines**](https://www.w3.org/WAI/standards-guidelines/wcag/) (WCAG) are part of a series of web accessibility guidelines published by the **[Web Accessibility Initiative](https://www.w3.org/WAI/about/)** (WAI) . The WCAGs goal is to provide a single shared standard for web content accessibility that meets the needs of individuals, organizations and governments internationally. 
+
+## WCAG 2.0
+
+WCAG 2.0 was published on 11 december 2008 and consists of 12 to 13 guidelines. It is organized around four principles which form the acronym `POUR`.  They include:
```

Comment:
WCAG is now at version 2.1, with version 2.2 due to be released shortly. Question for you: do you think we should use 2.1 in place of 2.0 here, or mention that while 2.0 was published in December of 2008 that version 2.1 is the current version that should be honored?

Reply:
To be honest, I think we should use 2.1

Reply:
Cool. Maybe after the first sentence you can have another sentence explaining that 2.1 is the current standard. I know you have some language later in the piece that goes into it, but I worry that people skimming the piece won't get that.

Reply:
Sounds great! I've done that